tutorials/02-logistic-regression/02_logistic-regression.jmd

```diff
@@ -233,7 +233,7 @@ println("Not defaults: $not_defaults
     Percentage non-defaults correct $(predicted_not_defaults/not_defaults)")
 ```
 
-```julia; echo=false
+```julia; echo=false; error=false
```

Comment:
Why were these assertions disabled again? The point of them is to verify the output of the notebook

Reply:
They are enabled. This option ensures that `weave` errors. Otherwise errors only end up in the output.